### ISSUE TITLE
[Bug Report] export types missing in latest version

### ISSUE THREAD

**Issue by tolking** [element-plus/element-plus#5324]
### Element Plus version
1.3.0-beta.4

### OS/Browsers version
-

### Vue version
3.2.26

### Reproduction Link
https://codesandbox.io/s/element-plus-spa-forked-0jr35?file=/src/main.ts

Can't reproduce online, need local

### Steps to reproduce
```ts
// in TypeScript
import { ElTag } from 'element-plus'
```

- [v1.3.0-beta.3](https://unpkg.com/browse/element-plus@1.3.0-beta.3/lib/index.d.ts)
- [v1.3.0-beta.4](https://unpkg.com/browse/element-plus@1.3.0-beta.4/lib/index.d.ts)

```diff
- export * from 'element-plus/es/components';
+ export * from '@element-plus/components';
```

Maybe the symbol `@` is causing the error

### What is Expected?
success

### What is actually happening?
Module '"element-plus"' has no exported member 'ElTag'. Did you mean to use 'import ElTag from "element-plus"' instead?

**Comment by kailong321200875** [comment]
I have the same problem

**Comment by ws511341206** [comment]
I have the same problem

**Comment by czq297297** [comment]
111

**Comment by JeremyWuuuuu** [comment]
I'll go check out the release script, might be something wrong with it.

**Comment by JeremyWuuuuu** [comment]
I think I have just found the issue, the patch is on the way. Sorry for the trouble it brought to you guys :pray:

**Comment by JeremyWuuuuu** [comment]
https://github.com/element-plus/element-plus/pull/5335 just got merged. We are about to release a new branch for this fix. Might take a while (~10min)

**Comment by JeremyWuuuuu** [comment]
Closing this due to the fix. Again, sincerely sorry for the issue that I brought to you. :bow:

**Comment by iamkun** [comment]
fixed in 1.3.0-beta.5
